### DISCUSSION TITLE
Last update crushes osu from the beggining

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#18924] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

Creepy

### Screenshots or videos

https://user-images.githubusercontent.com/86934170/176228024-f29e5187-f81d-49b3-86ad-125ff26b967f.mp4

### Version

628

### Logs

[network.log](https://github.com/ppy/osu/files/9002769/network.log)
[performance.log](https://github.com/ppy/osu/files/9002771/performance.log)
[updater.log](https://github.com/ppy/osu/files/9002772/updater.log)
[database.log](https://github.com/ppy/osu/files/9002773/database.log)
[runtime.log](https://github.com/ppy/osu/files/9002774/runtime.log)

**Comment by peppy** [comment]
Google suggests a virus scanner or otherwise may have corrupted the game. Please re-run the installer from https://github.com/ppy/osu#running-osu *without* uninstalling first.

**Reply by Loreos7** [reply]
Doesnt help

**Comment by bdach** [comment]
please update graphics device drivers and/or disable shadowplay if it is enabled.

**Comment by Loreos7** [comment, marked as answer]
I resolved the problem

**Reply by peppy** [reply]
How did you fix it?

**Reply by Loreos7** [reply]
I had to delete lazer completely. The thing is I tried to install this [release](https://github.com/ppy/osu/releases/tag/v2017.212.0) just for fun but unfortunately nothing happened, I couldn't even start the program. So I guess after lazer has updated to latest version some remaining files of 2017 release affected its launch.

**Reply by bdach** [reply]
please mention these sorts of details next time. would be better yet if you handled the problem you created for yourself from start to finish, but alas
